test/e2e/tests/autocomplete.test.ts

```diff
@@ -0,0 +1,8 @@
+import { runClassicTest } from "../runTest";
+
+it("Test autocomplete in the console.", async () => {
```

Comment:
Super nit, but can we either turn these `it(...)` statements into sentences, something like `it("autocompletes the console input")`, or use `test("autocomplete in the console")` (I think the latter probably makes more sense here personally).